_posts/2021-06-07-Synchronizing-physical-clocks-among-different-Subnero-modems.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Synchronizing physical clocks among different Subnero modems
```

Comment:
Do you really want to focus on Subnero modems only, rather than any UnetStack-based modem?

Reply:
OK. Updated the title now.

Reply:
You'd probably want to rename the file, and also ensure the main text is not too Subnero focused.

Reply:
But this is Subnero blog not Unetstack blog right? So does it matter?

Reply:
Ah, I didn't realize! Yes, if it's Subnero blog, that's perfectly fine. But which blog does this make most sense in?

Reply:
Since we were using `phy.clockCalib` which is specific to Subnero modems, we went with the decision to put this blog in Subnero. However, this parameter is also in Unet audio so, this content might be suitable for all Unet SDOAMs. I can move this content if needed. Let me know any suggestions.

Reply:
The `clockCalib` is specific to Yoda, not only to Subnero modems. But I don't have a strong preference one way or the other. Just having all technical blogs in one place makes it easier for readers to find.